python/eos/tasks.py

```diff
@@ -455,15 +455,15 @@ def sample_nested(analysis_file:str, posterior:str, base_directory:str='./', bou
     """
     analysis = analysis_file.analysis(posterior)
     results = analysis.sample_nested(bound=bound, nlive=nlive, dlogz=dlogz, maxiter=maxiter)
-    samples = _np.array(list(map(analysis._x_to_par, results.samples)))
+    samples = results.samples
```

Comment:
Ok, this is confusing. Why is this contained in a different commit? Please create three clean commits:
 - one that fixes the task ``sample_nested``
 - one that fixes the task ``corner_plot``
 - one that fixes the naming of the ``eos-analysis`` subcommand.